modules/virt-accessing-exported-vm-manifests.adoc

```diff
@@ -0,0 +1,133 @@
+    kind: VirtualMachine
+    name: example-vm
+  tokenSecretRef: example-token
+status:
+#...
+  links:
+    external:
+#...
+      manifests:
+      - type: all
+        url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/external/manifests/all <1>
+      - type: auth-header-secret
+        url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/external/manifests/secret <2>
+    internal:
+#...
+      manifests:
+      - type: all
+        url: https://virt-export-export-pvc.default.svc/internal/manifests/all <3>
+      - type: auth-header-secret
+        url: https://virt-export-export-pvc.default.svc/internal/manifests/secret
+  phase: Ready
+  serviceName: virt-export-example-export
+----
+<1> Contains the `VirtualMachine` manifest, `DataVolume` manifest, if present, and a `ConfigMap` manifest that contains the public certificate for the external URL's ingress or route.
+<2> Contains a secret containing a header that is compatible with Containerized Data Importer (CDI). The header contains a text version of the export token.
+<3> Contains the `VirtualMachine` manifest, `DataVolume` manifest, if present, and a `ConfigMap` manifest that contains the certificate for the internal URL's export server.
+
+. Log in to the target cluster.
+
+. Get the `Secret` manifest by running the following command:
++
+[source,terminal]
+----
+$ curl --cacert cacert.crt <secret_manifest_url> -H \ <1>
+"x-kubevirt-export-token:token_decode" -H \ <2>
+"Accept:application/yaml"
+----
+<1> Replace `<secret_manifest_url>` with an `auth-header-secret` URL from the `VirtualMachineExport` YAML output. 
+<2> Reference the `token_decode` file that you created earlier.
++
+For example:
++
+[source,terminal]
+----
+$ curl --cacert cacert.crt https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/external/manifests/secret -H "x-kubevirt-export-token:token_decode" -H "Accept:application/yaml"
+----
+
+. Get the manifests of `type: all`, such as the `ConfigMap` and `VirtualMachine` manifests, by running the following command:
++
+[source,terminal]
+----
+$ curl --cacert cacert.crt <all_manifest_url> -H \ <1>
+"x-kubevirt-export-token:token_decode" -H \ <2>
+"Accept:application/yaml"
+----
+<1> Replace `<all_manifest_url>` with a URL from the `VirtualMachineExport` YAML output.
+<2> Reference the `token_decode` file that you created earlier.
++
+For example:
++
+[source,terminal]
+----
+$ curl --cacert cacert.crt https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/external/manifests/all -H "x-kubevirt-export-token:token_decode" -H "Accept:application/yaml"
+----
+
+.Next steps
+
+* You can now create the `ConfigMap` and `VirtualMachine` objects on the target cluster by using the exported manifests.
```

Comment:
```suggestion
* You can now create the `ConfigMap` and `VirtualMachine` objects on the target cluster using the exported manifests.
```

Reply:
"by using" used to be an IBM Style Guide thing and I got into the habit of it. I think it adds clarity, so I might keep it.